cddl/corim-meta-map.cddl

```diff
@@ -1,4 +0,0 @@
-corim-meta-map = {
-  &(signer: 0) => corim-signer-map
-  ? &(signature-validity: 1) => validity-map
```

Comment:
I believe use of validity here is only as a hint to parsers who would have to parse the payload if a security relevant decision is being made as corim-meta-map is part of the COSE unprotected header. But given there may be implementations already that use corim-signer-map, maybe it is best to leave it as is? Note, that the freshness mechanism in corim (as is or as an extension to the corim-map) implies the hint in the unprotected header should parallel what is in the payload. Hence, corim-meta-map should be similarly extensible.  Given there may be existing use, leaving it as is and relying on profiles to define its use is less disruptive. I don't think collapsing corim-meta-map into corim-signer-map for simplification is that significant (esp. if the goal of the hint is to parallel what is included in the payload).